Philips.CodeAnalysis.Test/Maintainability/Maintainability/AvoidInvocationAsArgumentTest.cs

```diff
@@ -74,6 +80,37 @@ class Foo
   public string Do() {{ return ""hi"";}}
   public string Moo(string s) {{ return ""hi"";}}
 
+  public string MyTest()
+  {{
+    var s = Do();
+    return Moo(s);
+  }}
+}}
+";
+			VerifyCSharpDiagnostic(errorContent, DiagnosticResultHelper.Create(DiagnosticIds.AvoidInvocationAsArgument));
```

Comment:
This is a new Invocation as argument violation, right?

Reply:
Yes indeed! There are still a hodgepodge of miscellaneous findings, including a few argumentviolations that I didn't fix yet because their squiggles are "hidden" behind the missingcopyright analyzer that highlights the entire file.